api/src/main/java/org/hawkular/inventory/api/MultipleEntityBrowser.java

```diff
@@ -23,7 +23,7 @@
  * @author Lukas Krejci
  * @since 1.0
  */
-interface MultipleEntityBrowser<Entity> {
+public interface MultipleEntityBrowser<Entity> {
```

Comment:
This interface was supposed to be an internal detail of the API not really exposed to the end-users, but I can see a use for it the way you're using it in the tests.

Still I am not sure if it is advisable to expose it. While we can add a javadoc saying that this is not to be implemented on its own that is not the same as outright forbidding it by access rules.